src/sass/_gallery.scss

```diff
@@ -0,0 +1,28 @@
+.gallery {
+  height: 122px;
```

Comment:
height może lepiej na auto? ale to oczywiście tylko sugestia, może dobrze się skaluje, wiec tylko do ponownego rzucenia okiem